app/controllers/nonsipp/ReturnSubmittedController.scala

```diff
@@ -0,0 +1,155 @@
+      case None =>
+        val submissionDate = dateService.now()
+        for {
+          updatedUserAnswers <- Future.fromTry(request.userAnswers.set(ReturnSubmittedPage(srn), submissionDate))
+          _ <- saveService.save(updatedUserAnswers)
+        } yield submissionDate
+    }
+
+  private def getReturnPeriods(
+    srn: Srn
+  )(f: NonEmptyList[DateRange] => Future[Result])(implicit request: DataRequest[_]): Future[Result] = {
+    val accountingPeriods = request.userAnswers.list(AccountingPeriods(srn))
+
+    if (accountingPeriods.isEmpty) {
+      request.userAnswers.get(WhichTaxYearPage(srn)) match {
+        case Some(returnPeriod) => f(NonEmptyList.of(returnPeriod))
+        case None => Future.successful(Redirect(controllers.routes.JourneyRecoveryController.onPageLoad()))
+      }
+    } else {
+      f(NonEmptyList.fromListUnsafe(accountingPeriods)) // safe as empty check is performed
+    }
+  }
+}
+
+object ReturnSubmittedController {
+
+  def viewModel(
+    schemeName: String,
+    email: String,
+    returnPeriods: NonEmptyList[DateRange],
+    submissionDate: LocalDateTime,
+    pensionSchemeEnquiriesUrl: String,
+    managePensionSchemeDashboardUrl: String
+  ): SubmissionViewModel =
+    SubmissionViewModel(
+      "returnSubmitted.title",
+      "returnSubmitted.panel.heading",
+      "returnSubmitted.panel.content",
+      content = ParagraphMessage(Message("returnSubmitted.paragraph", email)) +
+        TableMessage(
+          NonEmptyList.of(
+            Message("returnSubmitted.table.field1") -> Message(schemeName),
+            Message("returnSubmitted.table.field2") -> returnPeriodsToMessage(returnPeriods),
+            Message("returnSubmitted.table.field3") -> Message(
+              "site.at",
+              submissionDate.format(DateRange.readableDateFormat),
+              submissionDate.format(DateRange.readableTimeFormat).toLowerCase()
+            )
+          )
+        ),
+      whatHappensNextContent =
+        ParagraphMessage("returnSubmitted.whatHappensNext.paragraph1") +
+          ParagraphMessage(
+            "returnSubmitted.whatHappensNext.paragraph2",
+            LinkMessage("returnSubmitted.whatHappensNext.paragraph2.link", pensionSchemeEnquiriesUrl)
+          ) +
+          ParagraphMessage("returnSubmitted.whatHappensNext.paragraph3") +
+          ListMessage.Bullet(
+            Message("returnSubmitted.whatHappensNext.list1") + LinkMessage(
+              Message("returnSubmitted.whatHappensNext.list1.link", schemeName),
+              managePensionSchemeDashboardUrl
+            ),
+            LinkMessage("returnSubmitted.whatHappensNext.list2", "javascript:window.print();")
+          )
+    )
+
+  private def returnPeriodsToMessage(returnPeriods: NonEmptyList[DateRange]): DisplayMessage = {
+    def toMessage(returnPeriod: DateRange): Message = Message(
+      "site.to",
+      returnPeriod.from.format(DateRange.readableDateFormat),
```

Comment:
Using show on a date will format it correctly for a view
```suggestion
      returnPeriod.from.show
```